core/pageObject/index.ts

```diff
@@ -0,0 +1,13 @@
+import { LoginPage } from "./loginPage";
+import { Dashboards } from "./dashbord";
+
+export const getPage = (pageName: string, page: any) => {
```

Comment:
You have a similar functionality in your BasePage now - getPage(). So this method here is redundant